app/src/main/java/com/sopt/kakaobank/presentation/MainActivity.kt

```diff
@@ -14,6 +16,46 @@ class MainActivity : BindingActivity<ActivityMainBinding>(R.layout.activity_main
     override fun initView() {
         // 로직 작성
         // exampleObserve()
+        initHomeFragment()
+        clickBottomNavigation()
+        addBadge()
+    }
+
+    private fun initHomeFragment(){
+        val currentFragment = supportFragmentManager.findFragmentById(binding.fcvHome.id)
+        if (currentFragment == null) {
+            supportFragmentManager.beginTransaction()
+                .add(binding.fcvHome.id, HomeFragment())
+                .commit()
+        }
+    }
+
+    private fun clickBottomNavigation() {
+        binding.bnvHome.setOnItemSelectedListener { item ->
+            when (item.itemId) {
+                R.id.menu_my_bank -> {
+                    replaceFragment(HomeFragment())
+                    true
+                }
+                else -> false
+            }
+        }
+    }
+
+    private fun replaceFragment(fragment: Fragment) {
+        supportFragmentManager.beginTransaction()
+            .replace(R.id.fcv_home, fragment)
+            .commit()
+    }
+
+    private fun addBadge() {
+        binding.bnvHome.getOrCreateBadge(R.id.menu_notification).apply {
+            isVisible = true
+        }
+
+        binding.bnvHome.getOrCreateBadge(R.id.menu_my_page).apply {
+            isVisible = true
+        }
```

Comment:
```suggestion
        binding.bnvHome.getOrCreateBadge(R.id.menu_notification).isVisible = true
        binding.bnvHome.getOrCreateBadge(R.id.menu_my_page).isVisible = true
```
apply를 적용한 이유가 따로 있을까요?